Week2/num8.js

```diff
@@ -0,0 +1,44 @@
+// 8
+
+// 8.1
+let fooString = "100";
+let fooNumber = 100 ;
+let fooArray = [1, 2, 3];
+let fooX ;
```

Comment:
I think the assignment wants you to use  a `boolean` here, since `undefined` is not a variable type, but `boolean` is. So could you change that?